packages/ai-jsx/src/react/index.ts

```diff
@@ -0,0 +1,84 @@
+import * as ReactModule from 'react';
+import * as LLMx from '../index.js';
+import { Serialize } from './serialize.js';
+export * from '../index.js';
+
+export declare namespace JSX {
+  type ElementType = ReactModule.JSX.ElementType | LLMx.JSX.ElementType | typeof React | typeof jsx;
+  type Element = ReactModule.JSX.Element & LLMx.Node;
+  type IntrinsicElements = ReactModule.JSX.IntrinsicElements;
+  type ElementChildrenAttribute = ReactModule.JSX.ElementChildrenAttribute & LLMx.JSX.ElementChildrenAttribute;
+}
```

Comment:
When people import this file, I wonder if it'll cause TS to now be more permissive than they perhaps want. Like, in vanilla React, you can't do:

```tsx
<ReactComponent />
<AIComponent />
```

Once you import our types here, you can. But it also lets you put AI Components where they don't belong, right?

Even if that's true, this is still a net win and something we should move forward with. But we might want to call out the caveat.